src/Symfony/Component/HttpKernel/Controller/TraceableArgumentResolver.php

```diff
@@ -35,9 +35,11 @@ public function getArguments(Request $request, callable $controller)
     {
         $e = $this->stopwatch->start('controller.get_arguments');
 
-        $ret = $this->resolver->getArguments($request, $controller);
-
-        $e->stop();
+        try {
+            $ret = $this->resolver->getArguments($request, $controller);
+        } finally {
+            $e->stop();
+        }
```

Comment:
The `$ret` variable is no longer necessary with `finally`.

```suggestion
        try {
            return $this->resolver->getArguments($request, $controller);
        } finally {
            $e->stop();
        }
```